### ISSUE TITLE
Not possible to have multiple EZAudioPlotGL's?

### ISSUE THREAD

**Issue by dxclancy** [syedhali/EZAudio#318]
I have two different EZAudioPlotGL's in my view.  This does not seem to work. I think it may be because EZAudioPlotGL does frame related stuff in the init/setup.

**Comment by magnus80a** [comment]
No, you can in my experience only have one and I believe it's even mentioned in the README.md
Use EZAudioPlot instead if you need more than one.

**Comment by dxclancy** [comment]
My mistake, I assumed they were the same, that EZAudioPlot was backed by EZAudioPlotGL, and didn't bother to try the other, just tried duplicating the outlet/view from the example app.  thanks.
